framework/src/org/checkerframework/framework/test/diagnostics/TestDiagnosticUtils.java

```diff
@@ -179,7 +179,8 @@ protected static TestDiagnostic fromPatternMatching(
         String trimmed = original;
         String filename = "";
         if (noMsgText) {
-            if (!trimmed.contains("unexpected Throwable")) {
+            if (!trimmed.contains("unexpected Throwable")
```

Comment:
Can you add a comment what is special about these two strings and how to decide whether to add others?